css-pseudo-4/Overview.bs

```diff
@@ -724,10 +725,24 @@ Default UA Styles</h3>
     /* Represent default spelling/grammar error styling in an adjustable way */
     :root::spelling-error { text-decoration-line: spelling-error; }
     :root::grammar-error  { text-decoration-line: grammar-error; }
-    /* Highlight targeted text */
-    :root::target-text    { color: MarkText; background: Mark; }
   </pre>
 
+  Some highlight pseudo-elements
+  should have <dfn>default highlight colors</dfn>,
+  consisting of a 'color' and a 'background-color'.
+  For ''::selection'' they are
+  ''HighlightText'' and ''Highlight'',
+  while for ''::target-text'' they are
+  ''MarkText'' and ''Mark''.
```

Comment:
Did we decide to add paired-cascading behavior to ''::target-text'' also? There's afaik no Web-compatibility requirement for it here, right?

Reply:
not formally, and not for web compat, just based on impl opinions from Emilio and i (<https://github.com/w3c/csswg-drafts/issues/6386#issuecomment-957389344>).

Reply:
s/are/should correspond to/ done in https://github.com/w3c/csswg-drafts/commit/4e092c568e8a4247309ffff98fae7b172f2b4067

Reply:
> just based on impl opinions from Emilio and i (https://github.com/w3c/csswg-drafts/issues/6386#issuecomment-957389344).

I think this deserves its own issue and WG resolution